03-transactions.md

```diff
@@ -86,14 +86,14 @@ This output sends funds back to the owner of this commitment transaction, thus m
         # Penalty transaction
         <revocation-pubkey>
```

Comment:
Hmm, I'll fix those too.  Was just aiming for actual defined field names, but that was kind of silly.